grommet/grommet
# Valid prop inside onValidate callback of Form was not considering the required case of Checkbox.

#### What does this PR do?
Addition to the PR (https://github.com/grommet/grommet/pull/4996), where checkbox's case check was missing

#### Where should the reviewer start?
Form.js

#### What testing has been done on this PR?
Tested on new stories covering different use cases & creating new test cases.

#### How should this be manually tested?
Submit button should be enabled in the story, 'validate on change' after passing validity checks.

#### Any background context you want to provide?
https://github.com/grommet/grommet/pull/4996

#### What are the relevant issues?
https://github.com/grommet/grommet/issues/5012

#### Do the grommet docs need to be updated?
No

#### Should this PR be mentioned in the release notes?
No

#### Is this change backwards compatible or is it a breaking change?
Not a breaking change
